doc/meetings/2023_03_15.md

```diff
@@ -0,0 +1,65 @@
+# Meeting 2023-03-15
```

Comment:
Maybe you could add here that it was the meeting for the polyculture focus group, since the focus group for gamification had their meeting on the exact same date. Same goes for the file name.

Reply:
https://github.com/ElektraInitiative/PermaplanT/pull/118/commits/7757ad73907f98fe455c9006dbbaf755733cd4f5